### DISCUSSION TITLE
empty trash

### DISCUSSION THREAD

**Discussion by adalbertomania** [0xJacky/nginx-ui#542] [Q&A] [answered]
I have users in the Manage Users List and they have also logins in trash, they can't login because the trash don't delete the old access. 
How to empty trash?

**Comment by adalbertomania** [comment, marked as answer]
ok database.db in /usr/local/etc/nginx-ui edited, sad the UI can't delete users.

**Reply by 0xJacky** [reply]
This is a bug, i will fix it later this week.

**Reply by adalbertomania** [reply]
> This is a bug, i will fix it later this week.

thank you chief, do it in the UI is great.
